examples/nrunner/recipes/runnables/requirement_asset.json

```diff
@@ -0,0 +1 @@
+{"kind": "requirement-asset", "kwargs": {"name": "gpl-2.0.txt", "locations": "https://mirrors.kernel.org/gnu/Licenses/gpl-2.0.txt"}}
```

Comment:
How should users define more than one location?

Reply:
Considering we are using JSON representation, lists were supposed to work out of the box: 

`{"type": "asset", "name": "gpl-2.0.txt", "locations": ["https://mirrors.kernel.org/gnu/Licenses", "http://mirrors.kernel.org/gnu/Licenses/"], "asset_hash": "4cc77b90af91e615a64ae04893fdffa7939db84c"}`

But looks like they are not working. I'm checking why.

Reply:
Multiple problems found :D

The first is fixed here: https://github.com/avocado-framework/avocado/pull/4652

Reply:
PR #4652 fixed the problem while running the tests with the state machine.

To run the runner in the command line, the following worked: `avocado-runner-requirement-asset runnable-run name=gpl-2.0.txt locations='json:["https://mirrors.kernel.org/gnu/Licenses", "http://mirrors.kernel.org/gnu/Licenses"]'`